spec/unit/routing/mapper_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
```

Comment:
** Code Quality - Nitpick**
Cover routing with unit specs in addition to integration specs.